trillian/ctfe/handlers.go

```diff
@@ -506,6 +506,11 @@ func getSTHConsistency(ctx context.Context, c *LogContext, w http.ResponseWriter
 			return c.toHTTPStatus(err), fmt.Errorf("backend GetConsistencyProof request failed: %v", err)
 		}
 
+		// We can get here with a tree size too small to satisfy the proof.
+		if rsp.GetSignedLogRoot() != nil && rsp.GetSignedLogRoot().TreeSize < second {
```

Comment:
Everywhere else uses direct accessors -- can we be consistent (ie. `rsp.SignedLogRoot`)?  (passim)

Reply:
Getters work with nil receivers so are potentially preferable. I think the Go proto library API changed since this code was written. For the moment I'll put them back to direct accessors.